tests/unit/test_ffmpeg.py

```diff
@@ -103,8 +103,8 @@ def test_frames(video_with_small, start, end, step, error):
         ff_frames(**arguments)
 
         # Check that progress updates are complete
-        expected_file_no = ((end - start) / step) + 1
-        assert frame_indices == range(1, expected_file_no + 1)
+        expected_file_no = int(((end - start) / step) + 1)
```

Comment:
So, either you cast both operands and then use integer division or you cast the final result (which is certain to have a zero floating part from the constraints of `ff_frames`).